Sources/Logging/LoggingFormatter.swift

```diff
@@ -0,0 +1,225 @@
+//
+//  LoggingFormatter.swift
+//  Logging
+//
+//  Created by Ivana Mršić on 24.05.2021..
+//  Idea taken from: https://github.com/SwiftyBeaver/SwiftyBeaver
+//
+
+import Foundation
+import os
+
+// MARK: - Print -
+
+enum LoggingFormatter {
+
+    /// Returns the current thread name.
+    static var threadName: String {
+        if Thread.isMainThread {
+            return ""
```

Comment:
Should we maybe specify "Main" here instead?

Reply:
In case if you set thread name (string is not empty), this will be printed in the console (empty string is skipped). As more than 90% of the codebase is logged from the main thread, I think that this would just add noise to the debugger so I would suggest not printing "Main" in the console if that is not needed.